### ISSUE TITLE
Orden de estímulos en entrenamiento y clasificación

### ISSUE THREAD

**Issue by lucasbaldezzari** [lucasbaldezzari/BCIC-Personal#21]
Atender y acomodar el orden de los estímulos (labels) durante la sesión de entrenamiento y para el clasificador. Deben corresponderse con el orden de _self.trainingSignalPSD_ dentro de _SVMClassifier_ y al mismo tiempo deben coincidir con la tabla de probabilidades _self.probab_.

**Comment by lucasbaldezzari** [comment]
Done!
